test/test_SimulatedChopper.py

```diff
@@ -19,8 +19,8 @@
 
 import unittest
 
-from lewis.devices.chopper.states import DefaultIdleState
-from lewis.devices.chopper import SimulatedChopper
+from lewis.devices.chopper.devices.states import DefaultIdleState
+from lewis.devices.chopper.devices.device import SimulatedChopper
```

Comment:
This looks a bit weird to me. Is there a better filename than `device`? It seems the word `device` is repeated a lot of times now.